### ISSUE TITLE
Expose build method on Blocks

### ISSUE THREAD

**Issue by hang15** [raycharius/slack-block-builder#22]
I only need to generate the Blocks Element, especially when using other frameworks that can already generate the "surface" payload.

**Comment by raycharius** [comment]
Thanks for the input! How would you see this working? Having a `Blocks` object at the top level with an `append` method? 

Do the `getBlocks()` methods on all of the Surfaces not solve the issues?

**Comment by raycharius** [comment]
Closed, as the `getBlocks()` currently solves the issue, not sure this really requires a new wrapper object.

**Comment by raycharius** [comment]
Proposed change:

A `Blocks` object that is a canvas for creating a collection of blocks, for cases when the surface configuration is handled elsewhere in the application.

``` typescript
import { Blocks } from 'slack-block-builder';

export default () => {
  return Blocks(
    // Add block elements
  ).buildToJSON();
}
```

**Comment by zwily** [comment]
One place this can be handy is for unfurls, that don't take a surface. Instead you send a map of ```
   [url]: blocks
```

**Comment by raycharius** [comment]
Enjoy! It is currently available via the v2.0.0 Beta.

Can read more about the implementation [here](https://github.com/raycharius/slack-block-builder#utility-functions).

``` bash
npm install --save slack-block-builder@2.0.0-beta.1
```
